### PR TITLE
Added support liveness and readiness for kyuubi pod

### PR DESCRIPTION
**Pull request by murong00**
### _Why are the changes needed?_
It is better to support `livenessProbe` and `readinessProbe` for indicating whether the container is running.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
cc @ulysses-you @hddong

File: docker/helm/values.yaml [hunk 1]
```diff
@@ -28,6 +28,18 @@ image:
...
+  liveness:
+    enabled: true
+    failureThreshold: 10
+    initialDelaySeconds: 30
+    periodSeconds: 10
+  readiness:
+    enabled: true
+    failureThreshold: 10
+    initialDelaySeconds: 30
+    periodSeconds: 10
```
Comment (ulysses-you) [inline docker/helm/values.yaml:41]: can we keep the same ordering with `kyuubi-depolyment.yaml` ?
```
initialDelaySeconds
periodSeconds
failureThreshold
```

Comment (murong00) [reply docker/helm/values.yaml:40]: @ulysses-you Done, thanks.

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Pull request closed without merging**